srcpkgs/efl/template

```diff
@@ -52,7 +52,9 @@ license="LGPL-2.1-only"
 homepage="https://enlightenment.org"
 changelog="https://git.enlightenment.org/core/efl.git/plain/NEWS"
 distfiles="https://download.enlightenment.org/rel/libs/efl/efl-${version}.tar.xz"
-checksum=2979cfbc728a1a1f72ad86c2467d861ed91e664d3f17ef03190fb5c5f405301c
+checksum=d9f83aa0fd9334f44deeb4e4952dc0e5144683afac786feebce6030951617d15
+
+# ERROR: efl-1.26.3_2: do_check: 'xvfb-run dbus-run-session ninja -C build test'
```

Comment:
this isn't a useful message. from the comment in `do_check()`, it sounds like these tests are flaky, so say something like that